projects/1.20/assets/ex-deorum/exdeorum/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+{
+    "advancements.exdeorum.core.barrel.description": "使用木桶将有机物堆肥以形成泥土",
+    "advancements.exdeorum.core.barrel.title": "那是绿色垃圾",
+    "advancements.exdeorum.core.crook.description": "制作一个钩子，用它从树叶上收获更多的树苗。",
```

```suggestion
    "advancements.exdeorum.core.crook.description": "合成一把钩子，用它从树叶上收获更多的树苗。",
```
craft 必须翻译为合成

原文存在 double sapling drops，实际游戏内容是双倍掉落概率还是双倍掉落？

---

我自己玩的时候感觉是双倍掉落概率所以翻译成了收获更多的树苗